code/game/machinery/computer/arcade/_arcade.dm

```diff
@@ -0,0 +1,104 @@
+/obj/machinery/computer/arcade
+	name = "\proper the arcade cabinet which shouldn't exist"
+	desc = "This arcade cabinet has no games installed, and in fact, should not exist. \
+		Report the location of this machine to your local diety."
+	icon_state = "arcade"
+	icon_keyboard = null
+	icon_screen = "invaders"
+	light_color = LIGHT_COLOR_GREEN
+	interaction_flags_machine = INTERACT_MACHINE_ALLOW_SILICON|INTERACT_MACHINE_REQUIRES_LITERACY
+
+	///Static list of radial menu options so we don't have to remake it every time.
+	var/static/list/radial_menu_options
```

Comment:
FWIW I don't think we gain much caching this since I don't anticipate many people will be resetting arcade machines, but in any case this doesn't need to be datum-level, you can just put this in the proc it's used since it's only used in one place (I think)